src/main/java/com/epam/spring/homework2/beans/MyBeanPostProcessor.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.spring.homework2.beans;
+
+import org.springframework.beans.factory.config.BeanPostProcessor;
+
+public class MyBeanPostProcessor implements BeanPostProcessor {
```

Comment:
This bean is missing `Component `annotation, that's why validation is not working, please add it and see if validation started to execute